benefit/data/sheets/CaseTypeTab/CaseTypeTab.json

```diff
@@ -265,5 +266,12 @@
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "language_sscs", "TabLabel": "Welsh", "TabDisplayOrder": 25, "CaseFieldID": "languagePreferenceWelsh", "TabFieldDisplayOrder": 1, "TabShowCondition": "languagePreferenceWelsh=\"Yes\""},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "language_sscs", "TabLabel": "Welsh", "TabDisplayOrder": 26, "CaseFieldID": "translationWorkOutstanding", "TabFieldDisplayOrder": 2},
   {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "hearingRecordingsInfo_sscs", "TabLabel": "Hearing Recordings", "TabDisplayOrder": 27, "CaseFieldID": "sscsHearingRecordings", "TabFieldDisplayOrder": 2},
-  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "other_party_sscs", "TabLabel": "Other Party Details", "TabDisplayOrder": 28, "CaseFieldID": "otherParties", "TabFieldDisplayOrder": 1}
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "other_party_sscs", "TabLabel": "Other Party Details", "TabDisplayOrder": 28, "CaseFieldID": "otherParties", "TabFieldDisplayOrder": 1},
+
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "TabID": "wa_fields", "TabLabel": "WA fields", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsInternal", "TabFieldDisplayOrder": 1},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNameHmctsRestricted", "TabFieldDisplayOrder": 2},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseNamePublic", "TabFieldDisplayOrder": 3},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "ogdType", "TabFieldDisplayOrder": 4},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "caseManagementCategory", "TabFieldDisplayOrder": 5},
+  {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "wa_fields", "TabLabel": "Appeal Details", "TabDisplayOrder": 29, "CaseFieldID": "CaseAccessCategory", "TabFieldDisplayOrder": 6}
```

Comment:
The tab label is "Appeal Details", but there is already a tab called that, but the tab id is wa_fields.
Is this meant to be a separate tab?

Reply:
Will add this as part of CAM